incubator/exporter-stdout/src/main/java/io/aklivity/zilla/runtime/exporter/stdout/internal/stream/TlsEventHandler.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.stdout.internal.stream;
+
+import java.io.PrintStream;
+import java.util.function.LongFunction;
+
+import org.agrona.DirectBuffer;
+
+import io.aklivity.zilla.runtime.exporter.stdout.internal.types.event.TlsEventFW;
+import io.aklivity.zilla.runtime.exporter.stdout.internal.types.event.TlsFailedEventFW;
+
+public class TlsEventHandler extends EventHandler
+{
+    private static final String TLS_FAILED_FORMAT =
+        "ERROR: TLS Failed [timestamp = %d] [traceId = 0x%016x] [binding = %s.%s] [error = %s]%n";
```

Comment:
Let's review the stdout output format for all the event types.

Reply:
sure